src/util/constants.rs

```diff
@@ -101,6 +101,10 @@ pub const BITS_IN_PAGE: usize = 1 << LOG_BITS_IN_PAGE;
 /* Assume byte-addressability */
 pub const LOG_BYTES_IN_ADDRESS_SPACE: u8 = BITS_IN_ADDRESS as u8;
 
+// TODO: Should this be VM specific?
+pub const LOG_MIN_OBJECT_SIZE: u8 = LOG_BYTES_IN_WORD;
```

Comment:
In the `SIDE_GC_BYTE_SPEC` case, the min object size mmtk can support is one word (two words is the minimum size mmtk can support in a specific combination of `HAS_GC_BYTE` and `GC_BYTE_OFFSET`).

However, allowing larger min object size for various VMs has benefits such as better side metadata space efficiency.
So, probably we should consider this as a future goal?

Reply:
Yes, but how do we use it in compile-time constants such as constant instances of `SideMetadataSpec`?

Reply:
Yeah, I see.

Reply:
Yes I think this should be VM-specific constant. To use it in compile-time constants, we need to have access to `VM` type when constructing `SideMetadataSpec`.

Reply:
`SideMetadataSpec` is a constant. I don't see a way to access `VM` when creating a constant.